exercises/space-age/.meta/additional_tests.json

```diff
@@ -0,0 +1,12 @@
+{
+  "cases": [
+    {
+      "description": "age in seconds",
+      "property": "SpaceAge",
```

Comment:
Do you mean

> "property": "age"

instead of 

> "property": "seconds",

"property" isn't used in the template, but "age" would make it consistent with `canonical-data.json`.

Reply:
This resolves with the decision to remove the additional test.